examples/iot-dashboard/iot-dashboard.60

```diff
@@ -409,19 +409,19 @@ BoxWithButtons := Box {
 
 export IndoorTemperature := BoxWithButtons {
     property <int> temperature: 24;
-    title_: "Indoor Temperature";
+    title-: "Indoor Temperature";
     iconFile: @image-url("images/indoor-temperature.png");
     value: (temperature < 0 ? "" : "+") + temperature;
     isBright: true;
     changed(delta) => { temperature += delta; }
 }
 export Humidity := BoxWithButtons {
-    property <int> humidity_percent : 30;
-    title_: "Humidity";
+    property <int> humidity-percent : 30;
+    title-: "Humidity";
```

Comment:
this looks a bit strange

Reply:
Ahh, because `Box` also has a `title` property and it's an error to override it :-(